tests/integration/telemetry/tracing/clienttracing/client_tracing_test.go

```diff
@@ -18,6 +18,7 @@ package client
 import (
 	"errors"
 	"fmt"
+	"istio.io/istio/pkg/test/framework/resource"
```

Comment:
The position of this new import looks weird to me, same happens to several other files.

Reply:
I thought we had or where putting in code changes to handle import ordering. Maybe we aren't testing PRs yet.

Reply:
we should... looking into it

Reply:
https://github.com/golangci/golangci-lint/issues/1351

Reply:
@stevenctl  Looks like this import and the one in tracing_test.go is still not fixed?

Reply:
Thanks